app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -451,10 +463,10 @@ class MainActivity : AppCompatActivity() {
     private fun setLineDetectionMode() {
         soundsPlayer.stop()
         VisionManager.setModelPerformanceConfig(
-                ModelPerformanceConfig.Separate(
-                        detectionPerformance = ModelPerformance.On(ModelPerformanceMode.FIXED, ModelPerformanceRate.LOW),
-                        segmentationPerformance = ModelPerformance.On(ModelPerformanceMode.FIXED, ModelPerformanceRate.HIGH)
-                )
```

Comment:
It is ok for now. But I think It would be better to format code as separate commit.

Reply:
Ok